google-cloud-logging/src/main/java/com/google/cloud/logging/v2/ConfigServiceV2Client.java

```diff
@@ -0,0 +1,2436 @@
+/*
```

Comment:
@hzyi-google Is this naming correct?

Reply:
Gapic config specified class renaming [here](https://github.com/googleapis/googleapis/blob/48a166ce9f6e2a88b10947341b37336fe9dd3478/google/logging/v2/logging_gapic.legacy.yaml#L5-L9). This regeneration deleted gapic config so the generator uses [proto service name](https://github.com/googleapis/googleapis/blob/48a166ce9f6e2a88b10947341b37336fe9dd3478/google/logging/v2/logging.proto#L40) to generate client class name.

If the change cause problems with manual code or if you think this change is very user-unfriendly we can add the interface name override back (very easy fix). Otherwise if logging needs to take a major bump anyway, we may just squash this change into it as we don't need to be bothered by gapic v2.